src/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -65,14 +106,39 @@ public function unPublishArticle($title)
 		$I->seeNumberOfElements(ContentListPage::$seeUnpublished, 1);
 	}
 
+	/**
+	 * Helper function to publish article
+	 *
+	 * @param   string  $title
+	 *
+	 * @since   4.0.0
+	 */
+	public function publishArticle($title)
+	{
+		$I = $this;
+		$I->amOnPage(ContentListPage::$url);
+		$I->waitForElement(ContentListPage::$filterSearch, TIMEOUT);
+		$I->searchForItem($title);
+		$I->checkAllResults();
+		$I->clickToolbarButton('publish');
+		$I->see('1 article published.', Administrator\AdminPage::$systemMessageContainer);
+	}
+
+	/**
+	 * Helper function to trash article
+	 *
+	 * @param   string  $title
+	 *
+	 * @since   4.0.0
+	 */
 	public function trashArticle($title)
 	{
 		$I = $this;
 		$I->amOnPage(ContentListPage::$url);
 		$I->waitForElement(ContentListPage::$filterSearch, TIMEOUT);
-		$this->articleManagerPage->haveItemUsingSearch($title);
+		$this->searchForItem($title);
+		$I->checkAllResults();
 		$I->clickToolbarButton('trash');
 		$I->searchForItem($title);
-		$I->dontSee($title);
```

Comment:
why is this being removed?

Reply:
You suggested me to replace $this with $I in one of the above comments